### PR TITLE
Added class Arrays (tests passed)

### PR DESCRIPTION
**Pull request by firebenderyt123**

### CODE REVIEW

File: src/arrays/Arrays.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          array_push($tags, $tag);
+        }
+      }
+    }
+    foreach ($arr as $tag => $array) {
+      sort($arr[$tag], SORT_STRING);
+      //var_dump($array);
+    }
+    ksort($arr);
+    //var_dump($arr);
```
Comment (azmentor) [inline src/arrays/Arrays.php]: Remove unnecessary comment

File: src/arrays/Arrays.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        else
+        {
+          $arr[$tag] = [$inp['name']];
+          array_push($tags, $tag);
+        }
+      }
+    }
+    foreach ($arr as $tag => $array) {
+      sort($arr[$tag], SORT_STRING);
+      //var_dump($array);
```
Comment (azmentor) [inline src/arrays/Arrays.php]: Remove unnecessary comment

File: src/arrays/Arrays.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      foreach ($inp['tags'] as $tag) {
+        if (in_array($tag, $tags))
+        {
+          array_push($arr[$tag], $inp['name']);
+        }
+        else
+        {
+          $arr[$tag] = [$inp['name']];
+          array_push($tags, $tag);
+        }
```
Comment (azmentor) [inline src/arrays/Arrays.php]: Please follow the PSR-12 recommendations in your code style

**Comment by firebenderyt123** [comment]
Arrays was edited

File: src/arrays/Arrays.php [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    public function groupByTag(array $input): array
+    {
+        $arr = [];
+        $tags = [];
+        foreach ($input as $inp) {
+            foreach ($inp['tags'] as $tag) {
+                if (in_array($tag, $tags)) {
+                    array_push($arr[$tag], $inp['name'])
```
Comment (azmentor) [inline src/arrays/Arrays.php]: You forgot ";". So, unfortunately, this code doesn't work and tests can't be passed.

**Pull request merged**